recipe/meta.yaml

```diff
@@ -22,7 +26,7 @@ source:
 # instead of `project(METIS)`.
 
 build:
-  number: 1006
+  number: 1000
```

Comment:
Keep this at `0`

Reply:
For context the 1000 came from a much older migration effort ( https://github.com/conda-forge/metis-feedstock/pull/19 ) that preceded a lot of the niceties we have in Conda & friends that we do today. Hence why it is not needed going forward

Reply:
Thanks! I wasn't sure if was needed or not. Set back to zero

Reply:
All good. Understand this can be confusing. Most recipes have long since reset the `build/number` to `0` (due to a version bump) so this doesn't come up that often any more